src/main/java/com/nha/abdm/wrapper/hip/hrp/dataTransfer/requests/helpers/DataStatusResponses.java

```diff
@@ -10,7 +10,8 @@
 @AllArgsConstructor
 @Data
 @Builder
-public class DataCareContexts {
+public class DataStatusResponses {
```

Comment:
can we rename all the classes prefixed with `Data` to have prefixed with HealthInformation like DataStatusResponses to HealthInformationResponse ..class name should not be plural, if required then can be formed in a list